client/src/components/history/History.js

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
-
-const History = () => (
-  <div className="content">
-    <div className="history">
-      <h1 className="heading__main">History</h1>
-      <p className="body-text">This is where logged in users can view their previous entries.</p>
-    </div>
-  </div>
-);
+import React, { Component } from 'react';
+
+import FilterBar from './FilterBar';
+import SavingsCard from './SavingsCard';
```

Comment:
SavingsCard displays total income, expenses, and calculates total savings. It appears under <FilterBar />